openmc/model/model.py

```diff
@@ -136,29 +169,19 @@ def run(self, **kwargs):
         Returns
         -------
         2-tuple of float
-            k_combined from the statepoint
+            Combined estimator of k-effective from the statepoint
 
         """
-
         self.export_to_xml()
 
         return_code = openmc.run(**kwargs)
 
         assert (return_code == 0), "OpenMC did not execute successfully"
 
-        statepoint_batches = self.settings.batches
+        n = self.settings.batches
         if self.settings.statepoint is not None:
             if 'batches' in self.settings.statepoint:
-                statepoint_batches = self.settings.statepoint['batches'][-1]
-        self.sp = \
-            openmc.StatePoint('statepoint.{}.h5'.format(statepoint_batches))
-
-        return self.sp.k_combined
-
-    def close(self):
-        """Close the statepoint and summary files
-        """
+                n = self.settings.statepoint['batches'][-1]
```

Comment:
Or the method can check if triggers are set and search for the last statepoint that way, but if there's another statepoint laying around in the directory with a higher batch number, it could cause problems. Anyway, probably not something to address as part of this PR